app/services/search_suggestion_populator_service.rb

```diff
@@ -1,16 +1,17 @@
 class SearchSuggestionPopulatorService
+  BATCH_SIZE = 5000
+
   def call
     SearchSuggestion.unrestrict_primary_key
     TimeMachine.now do
       suggestions = SuggestionsService.new.call
-      suggestions = suggestions.uniq { |suggestion| [suggestion[:id], suggestion[:value]] }
+      suggestions = suggestions.uniq { |suggestion| [suggestion[:id], suggestion[:type]] }
```

Comment:
These lines are not affected by the TimeMachine block, aren't they?
Maybe we can move it outside the `TimeMachine.now do`

Reply:
I could do. I don't know why I would, though? It feels unnaffected/potentially like it would negatively impact the readability of the block :thinking: . I actually thing I can probably remove this line, anyway

Reply:
Removed it